src/angular-app/bellows/shared/sound-player.component.html

```diff
@@ -7,5 +7,5 @@
     </span>
     <!--suppress HtmlFormInputWithoutLabel -->
     <!--The step value is very small to handle a bug encountered where the slider stops after its last full step, before reaching the end 2022-09-->
-    <input class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" step="0.0000000000000001" value="0">
+    <input id="slider" class="seek-slider" type="range" min="0" max="{{$ctrl.durationInSeconds()}}" step="0.0000000000000001" value="0">
```

Comment:
I'm hoping we don't need this extra `id`

Reply:
agreed

Reply:
👍got it! Made this change.